Tensorflow/02_BuildImage.ipynb

```diff
@@ -93,7 +97,136 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Below is the module for the Flask web application."
+    "The use of Azure Application Insights can provide information about your service with very little work on the developers behalf.  \n",
+    "\n",
+    "To make use of Azure Application Insights. Simply follow the instructions in the code block below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%%writefile flaskwebapp/applicationinsights.py\n",
+    "from app import app\n",
```

Comment:
So this file is just the import and then comments to be uncommented if the user wants? I would rather we point to instructions rather than creating redundant code.